core/src/io/Dataparser.cpp

```diff
@@ -1071,32 +1082,31 @@ namespace IO
             myfile.Read_String( ovf_data_representation, "# Begin: Data" );
             std::istringstream repr( ovf_data_representation );
             repr >> ovf_data_representation;

```

Comment:
One last question: why these two lines?
```
std::istringstream repr( ovf_data_representation );
repr >> ovf_data_representation;
```
It seems to me like extracting the first word of a sequence, but the contents of `ovf_data_representation` should already be just "binary" or "text" right?

Reply:
Actually what we are doing is:

- Before the call to `Read_String()` the `ovf_data_representation` is just an empty string. 
- The `Read_String()` will get whatever is on the line, after the `keyword` (in our case `"# Begin: Data"`), till `\n`.
- The trick is that now `ovf_data_represantation` may __not__ hold only the data representation because in case of binary representation we would get `"binary x"` (where `x` is 4 or 8) instead of only `"binary"`. 
- We turn the result which is stored in `ovf_data_represantation` to an `std::istringstream` and we stream its first word to `ovf_data_represantation`.
- If `ovf_data_represantation` is `"text"` the `std::istringstream` is empty and we are done. If it is `"binary"` then the `std::istringstream` is just holding the binary length that we are going to stream in to the variable `ovf_binary_length`.